common/src/main/java/com/wynntils/features/user/ChatTimestampFeature.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.StartDisabled;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@StartDisabled
```

Comment:
Why? This is useful to be enabled by default IMO

Reply:
this feature is disabled by default on 1.12 wynntils afaik so that is why I disabled it by default

Reply:
Oh. Well, I'll leave it to you I guess.